samples/Tests/CustomParsingTestUsingProgressBar.json

```diff
@@ -15,13 +15,15 @@
 					"horizontalAlignment": "center"
 				},
 				{
-					"type": "CustomIOSDelegate",
+					"type": "ProgressBar",
 					"payload": [
 									{
 									"red":187,
 									"green":160,
 									"blue":209,
-									"alpha":1
+									"alpha":1,
+                                    "progress":0.95,
+                                    "thickness":1
```

Comment:
spaces?

Reply:
thanks

---
In reply to: [251641777](https://github.com/Microsoft/AdaptiveCards/pull/2325#discussion_r251641777) [](ancestors = 251641777)